docs/generated/cli/create-nx-workspace.md

```diff
@@ -77,7 +77,7 @@ Workspace name (e.g. org name)
 
 Type: boolean
 
-Use Nx Cloud
+Setup distributed caching using Nx Cloud? (It's free and doesn't require registration.)
```

Comment:
Does this need a question mark and the explanation in parens? Maybe we could actually add a link to nx.app here?